pset-iterators.js

```diff
@@ -128,6 +152,14 @@ declared with const
 console.log('Problem 4:')
 
 // Add your code below this line
+const alivePresidents = []
+presidents.forEach(function(element){
```

Comment:
@rvaswani85 `.filter()` would have been the preferred method here - [see this approach](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-iterators.js#L152) for more details